power-platform/admin/dlp-resource-exemption.md

```diff
@@ -24,8 +24,7 @@ search.app:
 
 # DLP resource exemption
 
-You can exempt apps and flows that you trust from DLP policies by using [DLP resource exemption PowerShell cmdlets](powerapps-powershell.md#dlp-resource-exemption-cmdlets). 
-
+You can define a list of exempted resources (canvas app, cloud flow, or chat bot) that will not be evaluated by a DLP policy by using [DLP resource exemption PowerShell cmdlets](powerapps-powershell.md#dlp-resource-exemption-cmdlets). You can exempt up to 2,000 low-code assets per DLP policy.
```

Comment:
In the last sentence, can we replace "2,000 low-code assets" with "2,000 resources".

Reply:
Also, if we can also exempt desktop flows, let's also mention them in the first sentence as such: "(canvas app, cloud flow, desktop flow, or chat bot)".

Reply:
Fixed.